spec/ad_parser.spec.js

```diff
@@ -1,46 +1,57 @@
 import { _parseExtensions } from '../src/parser/ad_parser';
 import { parserUtils } from '../src/parser/parser_utils';
 import { getNodesFromXml } from './utils/utils';
-import { extensions } from './samples/extensions';
+import { adExtensions } from './samples/extensions';
+import { creativeExtensions } from './samples/extensions';
 
 describe('AdParser', function() {
   describe('Extensions', function() {
-    let parsedExts;
+    let parsedAdExts, parsedCreativeExts;
 
     beforeAll(() => {
       const extensionNodes = parserUtils.childrenByName(
-        getNodesFromXml(extensions),
+        getNodesFromXml(adExtensions),
         'Extension'
       );
-      parsedExts = _parseExtensions(extensionNodes);
+      parsedAdExts = _parseExtensions(extensionNodes);
+
+      const creativeExtensionNodes = parserUtils.childrenByName(
+        getNodesFromXml(creativeExtensions),
+        'CreativeExtension'
+      );
+      parsedCreativeExts = _parseExtensions(creativeExtensionNodes, 'Creative');
```

Comment:
I would move this to a new describe block `CreativeExtensions` and rename this as `AdExtensions`. Also as mentioned in the other comment, you could move the new mocha tests here instead.

Reply:
Moved to a new block.